triggers/service/auth/src/main/scala/com/daml/auth/middleware/api/Client.scala

```diff
@@ -178,18 +234,37 @@ class Client(config: Client.Config) {
       }
     } yield authorize
 
+  private def appendToUri(uri: Uri, path: Uri.Path, query: Uri.Query = Uri.Query.Empty): Uri = {
+    val newPath: Uri.Path = uri.path ++ path
+    val newQueryParams: Seq[(String, String)] = uri.query().toSeq ++ query.toSeq
```

Comment:
The main motivation here was to leave the configured URI as untouched as possible. Duplicates could occur if the user configures an `--auth` flag with query params that conflict with the auth middleware protocol. In that case both the original parameter and the new parameter would be present. In general, multiple parameters with the same key are allowed in URIs and their meaning is implementation defined. akka-http's `parameters` combinator seems to use the first parameter in case of duplicates (maybe we should instead combine in case of `claims`?). However, the auth middleware is meant to be exchangeable. I'm not sure we should define the handling of multiple query parameters in a user's auth middleware implementation at this point.